test/test_stieltjes.jl

```diff
@@ -68,3 +68,100 @@ end
     @test u[0.1] ≈ exp(0.1)/sqrt(0.1-0.1^2)
     @test (L * u)[0.5] ≈ -7.471469928754152 # Mathematica
 end
+
+#################################################
+# ∫f(x)g(x)(t-x)^a dx evaluation where f and g in Legendre
+#################################################
+
+@testset "dot() for PowerKernelPoint of Legendre" begin
+    @testset "Cached Legendre power law integral" begin
+        a = 2*rand(1)[1]
+        t = 1.0000000001
+        Acached = PowerLawIntegral(Legendre(),a,t)
+        @test size(Acached) == (∞,∞)
+        @test Acached[1:20,1:20] == pointwisecoeffmatrixdense(a,t,20)
+    end
+    @testset "PowKernelPoint dot evaluation finite" begin
+        # basis
+        P = Legendre()
+        x = axes(P,1)
+        # set 1, same length coefficient vectors
```

Comment:
Better to do  `@testset "set 1, same length coefficient vectors"` since then its easier to track down failures